torch/csrc/jit/codegen/cuda/ir_utils.h

```diff
@@ -109,6 +110,10 @@ auto filterByType(const ContainerType& inputs) {
   return filterByType<FilterType>(inputs.cbegin(), inputs.cend());
 }
 
+std::vector<int> normalizeOld2New(
```

Comment:
this helper deserves a comment explaining what it does (and where it's useful)

Reply:
Added a comment. The code itself is just extracted out of `broadcast`.

Reply:
can we factor out the common pieces between this and broadcast?

Reply:
`normalizeOld2New` is already the common code that `broadcast` and `transpose` need. It's just used to preprocess their axis remapping parameters.

Reply:
got it, thanks